plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -0,0 +1,818 @@
+
+        if self.have.get("device_exists"):
+            consolidated_data = self.compare_ap_cofig_with_inputdata(self.have["current_ap_config"])
+            if consolidated_data:
+                self.log('Final AP Configuration data to update {}'.format(str(consolidated_data)),
+                      "INFO")
+                task_response = self.update_ap_configuration(consolidated_data)
+                self.log('Task respoonse {}'.format(str(task_response)),"INFO")
+                config_updated = True
+            else:
+                # Accesspoint does not need update
+                self.msg = "AP - {0} does not need any update"\
+                    .format(self.have.get("current_ap_config").get("ap_name"))
+                self.log(self.msg, "INFO")
+                responses = {}
+                del self.payload["device_list"]
+                responses["accesspoints_updates"] = {"response": self.payload["device_config"]}
+                self.result['msg'] = self.msg
+                self.result["response"].append(responses)
+                self.result["skipped"] = True
+                return self
+        else:
+            #need to add the method for creation.
+            pass
+
+        if config_updated or config_created:
+            responses = {}
+            if task_response and isinstance(task_response, dict):
+                self.check_task_response_status(task_response, "task_intent", True).check_return_status()
+                if self.status == "success":
+                    self.result['changed'] = True
+                    responses["accesspoints_updates"] = {"response": task_response}
+                else:
+                    self.module.fail_json(msg="Unable to get Task Details.", response=task_response)
+
+            if config_updated:
+                self.msg = "AP Configuration - {0} Updated Successfully"\
+                    .format(self.have["current_ap_config"].get("ap_name"))
+                self.log(self.msg, "INFO")
+                self.result['msg'] = self.msg
+                self.result['response'].append(responses)
+        return self
+
+    def verify_diff_merged(self, config):
+        """
+        Verify the merged status(Creation/Updation) of AP configuration in Cisco Catalyst Center.
+        Args:
+            - self (object): An instance of a class used for interacting with Cisco Catalyst Center.
+            - config (dict): The configuration details to be verified.
+        Return:
+            - self (object): An instance of a class used for interacting with Cisco Catalyst Center.
+        Description:
+            This method checks the merged status of a configuration in Cisco Catalyst Center by retrieving the current state
+            (have) and desired state (want) of the configuration, logs the states, and validates whether the specified
+            AP exists in the Catalyst Center configuration.
+        """
+        time.sleep(20)
```

Comment:
Why so much time sleep...it will slow down the task execution in module.

Reply:
if i check ap config immediately after task success response, still not reflecting. after 10 second only reflecting. due to that verify getting failed